File: programmers/길찾기 게임/solve.js
function solution(nodeinfo) {
    const answer = [[],[]];
    const inodeinfo = [];
    for(const key in nodeinfo)
        inodeinfo.push([...nodeinfo[key],Number(key) + 1])
    inodeinfo.sort((a,b)=> a[0] - b[0]);
    const finder = (info) => {
        if (info.length) {
        let high = [0,-1,0];
        let highIdx = 0;
        for(let i =0; i < info.length; ++i){
            if(info[i][1] > high[1]) { 
                high = info[i];
                highIdx = i;
            }
        }
        console.log(high);
        answer[0].push(high[2]);
        finder(info.slice(0,highIdx));
        finder(info.slice(highIdx + 1));
        answer[1].push(high[2]);
    }}
    finder(inodeinfo);
    console.log(answer);
    return answer;
}

File: programmers/숫자게임/solve.cpp
#include <bits/stdc++.h>

using namespace std;

int solution(vector<int> A, vector<int> B) {
    int answer = 0;
    sort(A.begin(),A.end());
    sort(B.begin(),B.end());
    for(int i = 0, j = 0; i < B.size(); ++i) {
        if(A[j] - B[i] < 0) {
            ++answer;
            ++j;
        }
    }
    return answer;
}